dotCMS/src/main/java/com/dotcms/security/apps/AppsAPIImpl.java

```diff
@@ -68,26 +69,69 @@
             .setVisibility(PropertyAccessor.FIELD, Visibility.ANY)
             .findAndRegisterModules();
 
-    private String toJsonString(final AppSecrets object) throws DotDataException {
+    /*
```

Comment:
If this is not needed anymore can be deleted.

Reply:
I'm gonna need it soon.